SmartTutor/ContentModel/Lectures/KnowledgeNode.cs

```diff
@@ -9,9 +9,12 @@ public class KnowledgeNode
         public KnowledgeNodeType Type { get; private set; }
         public int LectureId { get; private set; }
         public List<LearningObjectSummary> LearningObjectSummaries { get; private set; }
+        public List<KnowledgeNode> Prerequisites;
+        //TODO: Remove if EF6 supports unidirectional many-to-many
+        public List<KnowledgeNode> PrerequisiteOf { get; set; }
 
```

Comment:
We talked about adding a "recap learning object summary" - essentially each knowledge node could mark a single learning object summary that would serve as the "recap" for the knowledge node. This recap LOSummary could then be retrieved by the recommender.